### ISSUE TITLE
Problems that can't be trained

### ISSUE THREAD

**Issue by Ajv234453** [uchiyama33/isic-2024-2nd-place#1]
Hello, I am trying to run your code, and I can't start training now, I have copied it below, I suspect it is the problem of prepare_data.py? I'll be happy if I get a reply

/opt/conda/lib/python3.10/site-packages/torchmetrics/utilities/prints.py:43: UserWarning: No positive samples in
targets, true positive value should be meaningless. Returning zero tensor in true positive score
  warnings.warn(*args, **kwargs)  # noqa: B028
/opt/conda/lib/python3.10/site-packages/lightning/pytorch/loops/fit_loop.py:310: The number of training batches (14) is
smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to
see logs for the training epoch.
Epoch 0/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0/14 0:00:00 • -:--:-- 0.00it/s v_num: mpfv val/loss: 0.677 val/auroc:
                                                                                  0.479 val/loss_target_meta: 0.676
                                                                                  val/loss_total: 0.879 val/pauc: 0.018
                                                                                  val/pauc_best: 0.018
